EIPS/eip-2015.md

```diff
@@ -0,0 +1,74 @@
+---
+eip: 2015
+title: Wallet Update Chain Method
```

Comment:
The other RPC related EIPs all had a title meantioning RPC or the method. I think that would be useful here.

Reply:
I've changed it, let me know what you think